### PR TITLE
added test resource manager

### PR DESCRIPTION
**Pull request by aneesh-joshi**
to be reviewed for correcting implementation (if required)

### CODE REVIEW

**Review by mtambos — CHANGES_REQUESTED** [review]
Please correct the coding style using flake8.
You should also use a ORM instead of writing the queries by hand. I like peewee.

Finally, there should be no code outside functions, unless it is absolutely necessary.

In the case of the tests, they should go into a unittest.TestCase class

File: MOCC/src/configurations/test_aneesh/db_setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import MySQLdb
```
Comment (mtambos) [inline MOCC/src/configurations/test_aneesh/db_setup.py]: make code flake8 compliant

Comment (mtambos) [reply MOCC/src/configurations/test_aneesh/db_setup.py]: use peewee instead of writing queries by hand

Comment (mtambos) [reply MOCC/src/configurations/test_aneesh/db_setup.py]: put everything inside functions and then use if __name__ == '__main__'

File: MOCC/src/configurations/test_aneesh/resource_manager_ds_setup.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import PyTango
```
Comment (mtambos) [inline MOCC/src/configurations/test_aneesh/resource_manager_ds_setup.py]: make code flake8 compliant
put everything inside functions and then use if name == 'main'

File: MOCC/src/configurations/test_aneesh/resource_manager_tester.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import PyTango
```
Comment (mtambos) [inline MOCC/src/configurations/test_aneesh/resource_manager_tester.py]: make code flake8 compliant
put everything inside functions and then use if name == 'main'

File: MOCC/src/configurations/test_aneesh/ResourceManagerDS.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+"""resource manager tango device server"""
```
Comment (mtambos) [inline MOCC/src/configurations/test_aneesh/ResourceManagerDS.py]: make code flake8 compliant

File: MOCC/src/configurations/Resource-Manager/resource_manager_ds_setup.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import PyTango
```
Comment (mtambos) [inline MOCC/src/configurations/Resource-Manager/resource_manager_ds_setup.py:1]: make code flake8 compliant
put everything inside functions and then use if name == 'main'

File: MOCC/src/configurations/Resource-Manager/TestResourceManager.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import unittest
```
Comment (mtambos) [inline MOCC/src/configurations/Resource-Manager/TestResourceManager.py:1]: make code flake8 compliant

File: MOCC/src/configurations/Resource-Manager/ResourceManagerDS.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
+"""
```
Comment (mtambos) [inline MOCC/src/configurations/Resource-Manager/ResourceManagerDS.py:1]: make code flake8 compliant

File: MOCC/src/configurations/test_aneesh/db_setup.py (hunk 1 above)
Comment (aneesh-joshi) [reply MOCC/src/configurations/test_aneesh/db_setup.py]: Sorry
This was an older version in which I was using MySQL.
An error while pushing caused it to still exist.
Have removed it now.

**Comment by aneesh-joshi** [comment]
Is this way of passing data between device servers (through strings) an acceptable method?

Is there any way to pass python objects?

**Comment by mtambos** [comment]
No there's no way of passing arbitrary objects between tango device servers.
That's something we're discussing with tango's team at the moment, but it's not in their near future plans.

**Pull request merged**